frontend/src/metabase/visualizations/components/ChartSettings/BaseChartSettings/BaseChartSettings.tsx

```diff
@@ -194,19 +192,26 @@ export const BaseChartSettings = ({
     <>
       <ChartSettingsMenu data-testid="chartsettings-sidebar">
         {showSectionPicker && (
-          <SectionContainer>
-            <Radio
-              value={chartSettingCurrentSection ?? undefined}
-              onChange={handleShowSection}
-              options={sectionNames}
-              optionNameFn={v => v}
-              optionValueFn={v => v}
-              optionKeyFn={v => v}
-              variant="underlined"
-            />
-          </SectionContainer>
+          <Tabs
+            radius={0}
+            defaultValue={chartSettingCurrentSection}
+            onTabChange={handleShowSection}
+            styles={{
+              tab: {
+                flexBasis: `${100 / maxTabsNumber}%`,
```

Comment:
we want to avoid section name position jumping. If you check "before" video from the description, you'll notice if we change viz type, number of section changes as well so tab width changes accordingly, that's why we stick to 4 tabs and `25%` value for `flex-basis`

Reply:
Makes sense :+1: 
It would make a good comment in code though.

Reply:
added a comment and dropped the var